apps/interactive-calibration/defaultConfig.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <opencv_storage>
 <charuco_dict>0</charuco_dict>
-<charuco_square_lenght>200</charuco_square_lenght>
+<charuco_square_length>200</charuco_square_length>
```

Comment:
not sure about changing a config file because it can lead to incompatibility issues

Reply:
I believe this compatibility code can handle migration:
```
if (readFromNode(reader["charuco_square_lenght"], mCapParams.charucoSquareLength))
    std::cout << "DEPRECATION: Parameter 'charuco_square_lenght' has been deprecated (typo). Use 'charuco_square_length' instead." << std::endl;
readFromNode(reader["charuco_square_length"], mCapParams.charucoSquareLength);
```

Reply:
@alalek @dkurt Thank you for your advice.

> I believe this compatibility code can handle migration

I'll fix this point.

Reply:
@atinfinity do you have progress on this?

Reply:
@asmorkalov Sorry for the late reply. I'll try to do this weekend.

Reply:
@asmorkalov I fixed this point. https://github.com/opencv/opencv/pull/16445/commits/db87044f4ec7d43aeb442bf0d48e032c1c25f814